### ISSUE TITLE
import GPUImage broken

### ISSUE THREAD

**Issue by stanchiang** [jcampbell05/Silkscreen#1]
hi i'm on Xcode 8.3 and on the lines `import GPUImage` i get a no such module error. Any ideas on the solution?

**Comment by stanchiang** [comment]
just tried reinstalling the pods but getting the problem
[!] The following pods are integrated into targets that do not have the same Swift version:

- GPUImage required by Silkscreen iPad (Swift 3.0), Silkscreen MacOS (Swift 2.3)

**Comment by stanchiang** [comment]
fixed - had to explicitly set to swift 3 in build settings

**Comment by jcampbell05** [comment]
@stanchiang :) could PR this back ?

**Comment by stanchiang** [comment]
would but now there's a problem with the auto layout constraints. let's look into this first :)

```
Silkscreen iPad[17374:813767] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x610000097e30 UIStackView:0x7f9823a0b860.height == 0.45*UIStackView:0x7f9820c081b0.height   (active)>",
    "<NSLayoutConstraint:0x60000009a220 'UISV-canvas-connection' UIStackView:0x7f9820c081b0.top == UIStackView:0x7f9823a0b860.top   (active)>",
    "<NSLayoutConstraint:0x60000009a270 'UISV-canvas-connection' V:[UIStackView:0x7f9823a0b860]-(0)-|   (active, names: '|':UIStackView:0x7f9820c081b0 )>",
    "<NSLayoutConstraint:0x600000099d20 'UIView-Encapsulated-Layout-Height' UIStackView:0x7f9820c081b0.height == 724   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x60000009a220 'UISV-canvas-connection' UIStackView:0x7f9820c081b0.top == UIStackView:0x7f9823a0b860.top   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
```

![simulator screen shot apr 2 2017 10 38 27 am](https://cloud.githubusercontent.com/assets/3621407/24589521/9a477e64-1790-11e7-9a25-b41840f7a1cf.png)

**Comment by stanchiang** [comment]
what's it supposed to look like?

**Comment by jcampbell05** [comment]
@stanchiang Something like this :) must have broken it porting it to the Mac
![simulator screen shot nov 1 2016 13 00 02](https://cloud.githubusercontent.com/assets/1484989/24589574/82d078c8-17d4-11e7-838e-7fd30da104b2.png)

**Comment by stanchiang** [comment]
ok so i started over, reinstalled pods, did the swift 3 conversion and manually updated syntax as the Xcode fix-its came up and this time i'm just getting a blank screen for the iPad version and for mac it says it can't find one of the files  `#import "GPUImageVideoCamera.h"` (more of a linking issue b/c the file is for sure there)

**Comment by stanchiang** [comment]
how about i fork the repo, upload my changes and then you can take a look?

**Comment by jcampbell05** [comment]
Yes please :) The mac version was more of an experiment and needs alot more work. This repo originated on the iPad so thats the more complited version

**Comment by stanchiang** [comment]
hmm i was hoping to play with a working iOS version. is there a specific commit i could recommend i jump into that predates the mac stuff?

**Comment by stanchiang** [comment]
btw did you build this based off https://github.com/tapharmonic/Learning-AV-Foundation :)

**Comment by jcampbell05** [comment]
@stanchiang Nope :) I didn't actually get around to implementing the editing stuff yet. Was looking if GPUImage would work.

Try commit b7c9384bd22c107d426afdc4ef3c3e1ede0c4727
